examples/tutorial/jupyter/execution/pandas_on_ray/local/exercise_2.ipynb

```diff
@@ -90,7 +88,7 @@
    "source": [
     "# [Optional] Download data locally. This may take a few minutes to download.\n",
     "# import urllib.request\n",
-    "# url_path = \"https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2021-01.csv\"\n",
```

Comment:
We can have this be the 1.8GB `dask-data` dataset because in https://github.com/modin-project/modin/blob/master/examples/tutorial/jupyter/execution/pandas_on_ray/test/test_notebooks.py#L57-L61, the line above is replace (commented out) and the smaller `modin-test` dataset is used instead.